management/test/clipper_manager_test.py

```diff
@@ -0,0 +1,267 @@
+import unittest
+import sys
+import os
+import json
+import time
+import requests
+from sklearn import svm
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0, os.path.abspath('%s/../' % cur_dir))
+import clipper_manager
+
```

Comment:
Can you add a comment at the top of the file briefly describing how these tests are run? Basically just say that at the beginning of each test suite a local Clipper instance is created and the tests issue various commands against the running Clipper instance and verify their results.

This is just so that it's documented that these tests start Docker containers and interact with them, as opposed to being pure Python tests.